app/src/components/features/rules/RuleBuilder/Header/ActionButtons/CreateRuleButton/index.js

```diff
@@ -117,6 +118,7 @@ const CreateRuleButton = ({
   const currentActionText = MODE === APP_CONSTANTS.RULE_EDITOR_CONFIG.MODES.EDIT ? "Save" : "Create";
 
   const handleBtnOnClick = async (saveType) => {
+    trackRuleSaveClicked(MODE);
```

Comment:
we were supposed to also add some attribute to track if this was from cmd+s right?

Reply:
They are already seeded in `saved` events